README.md

```diff
@@ -1,2 +1,15 @@
 # ocp-mig-test-data
 Ansible automation to load test-data of various sample applications to aid OCP Migration development efforts
+
+# Usage
+
+Copy config.yml.example to config.yml and set variables
+
+Export KUBECONFIG
```

Comment:
`export KUBECONFIG=~/.kube/config` Is where the kubeconfig normally is, other than that this looks good

Reply:
Fixed :)